### ISSUE TITLE
Group 'ur_manipulator' not found in model 'ur'.  Invalid move group name: `ur_manipulator`

### ISSUE THREAD

**Issue by Aruen24** [catmulti7/dual_ur_ros_2#5]
nodes_to_start = [move_group_node,
                    #   robot_control_node,
                      rviz_node, servo_node
                      ]
I open the rviz_node, servo_node run ros2 launch dual_ur5_moveit_config ur_moveit.launch.py

move_group-1] You can start planning now!
[move_group-1] 
[servo_node_main-3] [ERROR] [1730543724.119931848] [moveit_robot_model.robot_model]: Group 'ur_manipulator' not found in model 'ur'
[servo_node_main-3] [ERROR] [1730543724.119954602] [moveit_servo.servo_calcs]: Invalid move group name: `ur_manipulator`
[servo_node_main-3] terminate called after throwing an instance of 'std::runtime_error'
[servo_node_main-3]   what():  Invalid move group name
[ERROR] [servo_node_main-3]: process has died [pid 407939, exit code -6, cmd '/ROS2/IsaacSim-ros_workspaces/moveit_ws/install/moveit_servo/lib/moveit_servo/servo_node_main --ros-args --params-file /tmp/launch_params_h26cve6b --params-file /tmp/launch_params_4vz4i62b --params-file /tmp/launch_params_u16ocdtl'].

**Comment by catmulti7** [comment]
You should leave robot_control_node uncomment and comment `rviz_node, servo_node`
In this repo we use robot_control_node, which implement in robot_control package, rather than servo_node

**Comment by Aruen24** [comment]
> You should leave robot_control_node uncomment and comment `rviz_node, servo_node` In this repo we use robot_control_node, which implement in robot_control package, rather than servo_node

@catmulti7 so rviz_node, servo_node can not support in this repo?

**Comment by catmulti7** [comment]
@Aruen24 No, servo_node is unavailable. As for rviz, it require some setup and unfortunately I'm not familiar with rviz. If you create a runnable .rviz file, feel free to submit a pull request.
